lean-toolchain

```diff
@@ -0,0 +1 @@
+leanprover/lean4:nightly-2022-06-13
```

Comment:
Not picking `2022-05-29` is fine if it's not going to enter our graph of dependencies, but then you should probably pick something more recent then. I believe `2022-07-05` should work 👍🏼

Reply:
I'm using 6-13 to match with the mathport3 dependency